### PR TITLE
Explain downloaded image and container sizes in User Guide

### PR DESCRIPTION
**Pull request by yangyuqian**
**\- What I did**

Explain how to figure virtual sizes of images and containers in User Guide.

**\- How I did it**

Add a new section in docs/tutorials/dockerimages.md

**\- How to verify it**

Perform following commands and go to [the page](http://127.0.0.1:8000/engine/tutorials/dockerimages/#/check-sizes-of-images-and-containers) to see document:

```
git clone git@github.com:yangyuqian/docker.git
git checkout is_11160_explain_image_container_size_prediction
make docs
```

**\- Description for the changelog**

Add User Guide for checking disk usage of images and containers.

**\- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/tutorials/dockerimages.md
Comment (SvenDowideit) [inline docs/tutorials/dockerimages.md]: `An` :)

**Comment by SvenDowideit** [comment]
@yangyuqian can you please squash the commits?

**Comment by yangyuqian** [comment]
@SvenDowideit Thanks, just fixed the typo and squashed commits.

**Comment by yangyuqian** [comment]
@SvenDowideit Thanks for the instructions, done.

File: docs/tutorials/dockerimages.md
Comment (thaJeztah) [inline docs/tutorials/dockerimages.md]: "on top of" (no "the")

**Comment by thaJeztah** [comment]
ping @mstanleyjones ptal

@yangyuqian also looks like this PR contains a merge commit, so needs a rebase (if you need help doing so; our contributors guide contains some instructions https://docs.docker.com/opensource/workflow/work-issue/#/pull-and-rebase-frequently)

**Review by mdlinville — APPROVED** [review]
LGTM after addressing the merge commit and the grammar error pointed out by @thaJeztah. If you can't get this done by tomorrow Sep 27, this PR will need to be closed and reopened against the unified Docs repo.

**Comment by mstanleyjones** [comment]
Correction, September 28.

File: docs/tutorials/dockerimages.md
Comment (yangyuqian) [inline docs/tutorials/dockerimages.md]: Sorry for the late response! I'll handle it today!

**Comment by yangyuqian** [comment]
Thanks, just merged commits and resolved the grammar error.

**Comment by SvenDowideit** [comment]
LGTM

**Review by mdlinville — APPROVED** [review]
LGTM for master since we have not migrated master to the new repo yet, but won't be able to go into 1.12. If you want it both places you will need to be patient with us and re-do the PR against the new docs repo. Let me know how you want to handle it.

**Comment by yangyuqian** [comment]
@mstanleyjones i see, i'll redo the PR against the new docs repo.

**Comment by thaJeztah** [comment]
Thanks @yangyuqian! I'll close the PR in this repo

**Pull request closed without merging**
